src/main/java/ro/developmentfactory/thedrones/controller/DroneStatusController.java

```diff
@@ -0,0 +1,42 @@
+package ro.developmentfactory.thedrones.controller;
+
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import ro.developmentfactory.thedrones.entity.DroneStatus;
+import ro.developmentfactory.thedrones.service.DroneStatusService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/dronesStatus")
```

Comment:
I liked how you designed the API in general! for the status it might be a good idea to have `@RequestMapping("/drones/status")`. we usually treat the path as resources, and this way we can tell that status is nested under the drone, so it belongs to a drone. that would be a bit more RESTful

Reply:
done